Packs/MalwareBazaar/Integrations/MalwareBazaar/MalwareBazaar_test.py

```diff
@@ -0,0 +1,221 @@
+import pytest
+
+import MalwareBazaar
+import CommonServerPython
+import io
+import json
+
+BASE_URL = 'https://test.com'
+
+
+def util_load_json(path):
+    """
+
+    Args:
+        path:
+
+    Returns:
+        json object read from the path given
+    """
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def create_client():
+    return MalwareBazaar.Client(
+        server_url=BASE_URL,
+        verify=False,
+        proxy=False,
+        headers={},
+        api_key="111"
+    )
+
+
+def test_dbot_score():
+    """
+
+    Given:
+        - response from MalwareBazaar on hash file
+
+    When:
+        - Running a file command
+
+    Then:
+        - creating a CommanResult containing DbotScore and relationship
```

Comment:
```suggestion
        - Make sure a CommandResult containing DbotScore with a BAD score and a relationship is returned.
```
also let's check the relationship